src/Bicep.Core/Semantics/Namespaces/SystemNamespaceType.cs

```diff
@@ -998,6 +998,7 @@ maxLength switch
                     .WithRequiredParameter("variableName", LanguageConstants.String, "Environment Variable Name.")
                     .WithReturnResultBuilder(ReadEnvironmentVariableResultBuilder, LanguageConstants.String)
                     .WithFlags(FunctionFlags.GenerateIntermediateVariableAlways)
+                    .WithOptionalParameter("default",LanguageConstants.String, "Default value to return if environment variable was not found.")
```

Comment:
if environment variable was not found.
=>
if environment variable is not found.

Reply:
QUESTION: What happens if the env var is found but has an empty value?

Reply:
you mean empty string/ white spaces? then the returned value will be the empty string

Reply:
not positive if there's a difference between a null value vs an empty value.

Reply:
from what I'm reading it sounds like best to not distinguish between those two.